script/prerender/prerender.tsx

```diff
@@ -8,12 +8,14 @@ import {
   REACT_APP_ARCHIVE_URL,
   REACT_APP_HIGHLIGHTS_URL,
   REACT_APP_OS_WEB_API_URL,
+  REACT_APP_PRACTICE_QUESTIONS_URL,
```

Comment:
there is no need to pretend there will be a url here, we won't be using it for a while. as long as the loader exists it won't be hard to add it later